cordformation/src/main/kotlin/net/corda/plugins/Node.kt

```diff
@@ -143,6 +146,22 @@ open class Node @Inject constructor(private val project: Project) {
         @Optional @Input get
         private set
 
+    // Should the cordform set up run data base migration scripts after installation - defaults to false for compatibility
+    // with current and previous Corda versions
+    @get:Optional
+    @get:Input
+    var runSchemaMigration: Property<Boolean> = project.objects.property(Boolean::class.javaObjectType).convention(false)
```

Comment:
Use `val` instead of `var`.

Reply:
I changed it to `val` locally - works exactly the same.
Still refuses to accept `runSchemaMigration = true`.